client/src/components/Chat/Menus/PresetsMenu.tsx

```diff
@@ -27,7 +27,7 @@ const PresetsMenu: FC = () => {
       <Trigger asChild>
         <button
           className={cn(
-            'pointer-cursor relative flex flex-col rounded-md border border-gray-100 bg-white text-left focus:ring-0 focus:ring-offset-0 dark:border-gray-700 dark:bg-gray-800 sm:text-sm',
+            'pointer-cursor relative flex flex-col rounded-md border border-gray-100 bg-white text-left dark:border-gray-700 dark:bg-gray-800 sm:text-sm',
```

Comment:
Any time you touch elements with legacy colors, please apply theme classnames, especially don't add legacy aka `*-gray-*`